common/unsafe/src/test/java/org/apache/spark/unsafe/PlatformUtilSuite.java

```diff
@@ -58,4 +61,17 @@ public void overlappingCopyMemory() {
       Assert.assertEquals((byte)i, data[i + 1]);
     }
   }
+
+  @Test
+  public void memoryDebugFillEnabledInTest() {
+    Assert.assertTrue(MemoryAllocator.MEMORY_DEBUG_FILL_ENABLED);
```

Comment:
Yea I think the problem is that Maven doesn't have the property set.

cc @ericl  we need to set that in Maven too.